charts/datadog/templates/agent-psp.yaml

```
@@ -17,6 +17,11 @@ metadata:
 spec:
   privileged: {{ .Values.agents.podSecurity.privileged }}
   hostNetwork: {{ .Values.agents.useHostNetwork }}
+  {{- if .Values.datadog.dogstatsd.useHostPort }}
```

In fact we also need that for APM, so we can cover 8125-8126 is any of these two features is activated

```suggestion
  {{- if or .Values.datadog.dogstatsd.useHostPort .Values.datadog.apm.enabled }}
    hostPorts:
  - min: 8125
    max: 8126
```

---

Thanks, I have applied the suggestion